src/Components/Replication/ReplicationSettingsForm.tsx

```diff
@@ -411,94 +506,123 @@ export default class ReplicationSettingsFormReplication extends React.Component<
           <Form.Item
             label={
               <span>
-                Filter Records&nbsp;
-                <Tooltip title="Set the labels a record should include/exclude to be replicated. If empty, all records are replicated.">
+                When Condition&nbsp;
+                <Tooltip title="JSON condition for filtering records based on labels. Example: {'&temperature': {'$gt': 25}, '&humidity': {'$lt': 60}}. Use & for labels and $ for operators. Supported operators: $gt, $gte, $lt, $lte, $eq, $ne, $and, $or">
```

Comment:
link to documentation instead of count all the supported operators, they will be extended all the time.